test/client_bandit_test.py

```diff
@@ -91,7 +92,7 @@ def test_get_bandit_action_bandit_does_not_exist():
         "nonexistent_bandit",
         "subject_key",
         DEFAULT_SUBJECT_ATTRIBUTES,
-        [],
```

Comment:
Did not catch that this test was broken because `is_graceful_mode` was set to True